Orange/widgets/data/owcolor.py

```diff
@@ -350,6 +350,7 @@ def set_data(self, data):
                                         self._create_proxies(domain.metas))
             self.openContext(data)
             self.data = data.transform(domain)
+            self.data.domain = domain
```

Comment:
@astaric but shouldn't then we fix `Table.from_table` as well to prevent `from_table` and `transform` to behave differently? Isn't `data.transform` currently just a syntactic sugar?